From e4ec31897a863ce79dd0fcdfa40dd9d0a378de9d Mon Sep 17 00:00:00 2001
From: ran shalit <rshalit@habana.ai>
Date: Tue, 17 Jan 2023 10:54:19 +0200
Subject: [PATCH] [SW-112281] board: greco: post-image: use cmn hdr

Use common header in emb-specs with the absolute offsets
of memory sections for creating the fit, instead of using
internal harcoded values inside the post-image-fit.sh
script. In this way we reduce the duplication.

Change-Id: I0743b9e5158807a9dec049e9b9ea63baa889efae
Signed-off-by: ran shalit <rshalit@habana.ai>
---

diff --git a/board/habanalabs/greco/post-image-fit.sh b/board/habanalabs/greco/post-image-fit.sh
index 091d652..168c9d8 100755
--- a/board/habanalabs/greco/post-image-fit.sh
+++ b/board/habanalabs/greco/post-image-fit.sh
@@ -11,19 +11,8 @@
 # load addresses in ITS file
 UBOOT_ADDR_LO=0xF0100000
 UBOOT_ADDR_HI=0x0000007F
-ARC_MGMT_DCCM_HI=0x0000007F
-ARC_MGMT_DCCM_LO=0xFB030000
-ARC_MGMT_ICCM_HI=0x0000007F
-ARC_MGMT_ICCM_LO=0xFB020000
 
 UBOOT_ZEPHYR_ITS_EXT=_sram
-ARC_MGMT_SRAM_HI=0x0000007F
-ARC_MGMT_SRAM_LO=0xFBFE4000
-
-ARC_PID_DCCM_HI=0x0000007F
-ARC_PID_DCCM_LO=0xFB010000
-ARC_PID_ICCM_HI=0x0000007F
-ARC_PID_ICCM_LO=0xFB000000
 
 IMG_PTR_OFFSET=0x00060000
 PREBOOT_PRI_OFFSET=0x00020000
